File: backend/src/main/java/br/ufpr/sept/so2/modules/iam/api/dto/RedefinirSenhaRequest.java
package br.ufpr.sept.so2.modules.iam.api.dto;

import jakarta.validation.constraints.NotBlank;

public record RedefinirSenhaRequest(
        @NotBlank(message = "Token obrigatório") String token,
        @NotBlank(message = "Informe a nova senha") String novaSenha
) {
}

File: backend/src/main/kotlin/br/ufpr/sept/so2/modules/iam/api/dto/RedefinirSenhaRequest.kt
package br.ufpr.sept.so2.modules.iam.api.dto

import jakarta.validation.constraints.NotBlank

data class RedefinirSenhaRequest(
    @field:NotBlank(message = "Token obrigatório")
    val token: String,
    @field:NotBlank(message = "Informe a nova senha")
    val novaSenha: String,
)
